src/app/recipes/store/recipe.effect.ts

```diff
@@ -42,6 +51,50 @@ export class RecipeEffects {
     })
   );
 
+  @Effect()
+  AddRecipeToDataBase = this.actions$.pipe(
```

Comment:
You should create variables from lowerCase (`addRecipeToDataBase`)

Reply:
fixed